index.html

```diff
@@ -57,27 +61,27 @@ <h2>Here's a time line of Dr. Borlaug's life:
           <li><b>2009</b> - dies at the age of 95.</li>
```

Comment:
Replace b tag with strong tag

Reply:
- [ ] Same here Kindly ask for it politely and mention the reason.